bika/lims/browser/widgets/reflexrulewidget.py

```diff
@@ -427,7 +435,17 @@ def getActionVoc(self):
         return DisplayList([
             ('repeat', 'Repeat'),
             ('duplicate', 'Duplicate'),
-            ('setresult', 'Set result')])
+            ('setresult', 'Set result'),
+            ('setvisibility', 'Set Visibility')])
+
+    def getShowInRepVoc(self):
+        """
+        Return the different Visibility in Report values.
+        """
+        return DisplayList([
+            ('default', 'Visibility (default)'),
+            ('visible', 'Show in Report'),
+            ('invisible', 'Hide In Report')])
```

Comment:
should be:

```python
return DisplayList([
    ('default', _('Visibility (default)')),
    ('visible', _('Show in Report'))),
    ('invisible', _('Hide In Report')))])
```
Otherwise, these strings will never appear translated.